src/EnergyPlus/DesiccantDehumidifiers.cc

```diff
@@ -188,6 +188,8 @@ namespace DesiccantDehumidifiers {
 	int NumSolidDesicDehums; // number of solid desiccant dehumidifiers
 	int NumGenericDesicDehums; // number of generic desiccant dehumidifiers
 	Real64 TempSteamIn( 100.0 ); // steam coil steam inlet temperature
+	static bool GetInputFlag( true ); // First time, input is "gotten"
+	static bool MyOneTimeFlag( true );
```

Comment:
When MyOneTimeFlag is pulled up out of a local function, it needs to have a more specific name to avoid confusion with other MyOneTimeFlag occurences.  Suggest using InitDesiccantDehumidifierOneTimeFlag here (and other corresponding places).